apps/picsa-tools/option-tool/src/app/components/editor/editor.component.ts

```diff
@@ -1,6 +1,15 @@
 import {Component,EventEmitter, OnInit, Output ,ViewChild} from '@angular/core';
 import { MatStepper } from '@angular/material/stepper'
 
+export interface IOptionData {
```

Comment:
Nice idea to store everything in a single data object, I think it works well and makes a lot of sense to me

Reply:
Yes sure that would be better